stable.txt

```diff
@@ -1 +1 @@
-v1.14.0
+v1.14.1
```

Comment:
I'm guessing @joestringer's [comment](https://github.com/cilium/cilium/pull/27637/files#r1302236122) ☝🏻 applies here too.

Reply:
This is correct. v1.14.1 is the latest release, it's the bit about v1.14.1-rc.1 which has no corresponding release.